### DISCUSSION TITLE
Groups should allow adding existing windows

### DISCUSSION THREAD

**Discussion by txtyash** [hyprwm/Hyprland#2608] [General]
Right now only newly spawned windows get added to the window group in the dwindle layout but pre existing windows can be added to the group with a nice behavior

Let's say I have a grouped window and on its left another window I'd like to add to the group then I should be able to do it with something like `mod+shift+h`, similarly to merge a window on the right of the group press `mod+shift+l`. This behavior makes grouping convenient and useful. I used to use it in xmonad.

**Comment by MightyPlaza** [comment]
Samerino
